packages/CodingStyle/src/Rector/ClassMethod/AnnotateThrowables.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\ClassMethod;
+
+use PhpParser\Comment\Doc;
+use PhpParser\Node;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Throw_;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+/**
+ * Adds "throws" DocBlock to methods.
+ */
+final class AnnotateThrowables extends AbstractRector
+{
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes() : array
+    {
+        return [Throw_::class];
+    }
+
+    /**
+     * @param Node|Throw_ $node
+     *
+     * @return Node|null
```

Comment:
```suggestion
```

Reply:
PR and code changed: please annotate the new PR.

Reply:
This is fixed by `composer fix-cs`

Reply:
https://github.com/rectorphp/rector/blob/a0f303f1b8397446824c14962e3e0507cac1ccb2/composer.json#L226